cif_pow.dic

```diff
@@ -969,15 +1013,66 @@ save_pd_calib_std.external_block_id
 
 save_
 
+save_pd_calib_std.external_k_factor
+
+    _definition.id                '_pd_calib_std.external_k_factor'
+    _definition.update            2022-12-01
+    _description.text
+;
+    The value of the K factor applied to the quantification of
+    the phases present in the given diffractogram.
+
+    The external standard method is described by O'Connor and Raven. In this
+    method, the absolute mass percent of a phase is given as
+
+    W_k = 100 * s_k * M_k * V_k * \mu* / K
+
+    where W_k, s_k, M_k, & V_k are the absolute mass percent, Rietveld
+    scale factor, unit cell mass, and unit cell volume of phase k.
+    \mu* is the mass attenuation coefficient of the specimen, and K is
+    the normalising diffractometer constant.
+
+    O'Connor, B. H., & Raven, M. D. (1988). Application of the Rietveld
+    Refinement Procedure in Assaying Powdered Mixtures. Powder Diffraction,
+    3(1), 2-6. doi:10.1017/s0885715600013026
+;
+    _name.category_id             pd_calib_std
+    _name.object_id               external_k_factor
+    _type.purpose                 Measurand
+    _type.source                  Derived
+    _type.container               Single
+    _type.contents                Real
+    _units.code                   none
+
+save_
+
+save_pd_calib_std.external_k_factor_su
+
+    _definition.id                '_pd_calib_std.external_k_factor_su'
+    _definition.update            2022-12-01
+    _description.text
+;
+    Standard uncertainty of _pd_calib_std.external_k_factor.
+;
+    _name.category_id             pd_calib_std
+    _name.object_id               external_k_factor
+    _name.linked_item_id          '_pd_calib_std.external_k_factor'
+    _units.code                   none
+
+    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
+
+save_
+
 save_pd_calib_std.external_name
 
     _definition.id                '_pd_calib_std.external_name'
     _alias.definition_id          '_pd_calib_std_external_name'
-    _definition.update            2016-10-18
+    _definition.update            2022-12-01
     _description.text
 ;
     Identifies the name of the material used as an external standard for
-    the diffraction angle or the intensity calibrations.
+    the diffraction angle or the intensity calibrations. This should
+    match the value given in _pd_phase.name.
```

Comment:
I think this means "_pd_phase.name" in the block containing the external standard information?

Reply:
Yes. I've clarified this.